osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -124,9 +125,24 @@ public abstract partial class SongSelect : ScreenWithBeatmapBackground, IKeyBind
         [Resolved]
         internal IOverlayManager? OverlayManager { get; private set; }
 
+        private Bindable<bool> backgroundBlurLevel { get; set; } = new BindableBool();
```

Comment:
variable needs a rename, probably `configBackgroundBlur`.

Reply:
both fixed in [e0a7559](https://github.com/ppy/osu/pull/22382/commits/e0a7559d851fca88f92d8866cb54a08f8cd0f6b0)